deployment/chainloop/templates/cas/configmap.yaml

```diff
@@ -21,7 +21,7 @@ data:
         # grpc downloads/uploads don't require this because they don't have timeouts
         timeout: 300s
       grpc:
-        {{- if .Values.cas.tlsConfig.secret.name  }}
+        {{- if .Values.cas.tls.existingSecret  }}
```

Comment:
I am worried this is a breaking change. We have users that are using this option today to enable e2e tls encryption. 

Could we support both options? 

Basically document the new one but if the user has the other one in the values.yaml file also render these?

Reply:
If would then personally won't add this option since it's just naming convention by Bitnami to be aligned with their chart which result in the same functionality with different naming.

Reply:
Right, but what if a user wants to start using a Bitnami Chart, e.g., because of VAC? I'd give them an easy ramp.  

It doesn't seem hard to me to create two helpers, one called `cas.tls-enabled` that makes an `or of those two conditions` and another one called `cas-tls-secret-name` or smth like that?